src/useLastLocation.spec.tsx

```diff
@@ -1,137 +1,68 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
which dependencies are extraneous?

Reply:
TS was throwing some weird issue with where the packages are being kept

Reply:
Yeah, if we can fix the TS stuff, that would be great. Didn't want to worry about it for now though...